src/debugpy/server/cli.py

```diff
@@ -43,7 +43,7 @@ class Options(object):
     address = None
     log_to = None
     log_to_stderr = False
-    target = None
+    target = None  # Unicode on Py2/Str on Py3
```

Comment:
Can just mark it as `unicode`, since we have that as an alias in `compat` on all Python versions.